src/containers/Application.js

```diff
@@ -221,6 +221,66 @@ it is accessible by the sidebar toggler button.
 	}
 
 
+	getWebSocketURL(service, subpath) {
+		// Handle if service is unknown => return undefined
```

Comment:
Can we start with:

```
		var service_url = this.getServiceURL(service);
		if (service_url == undefined) {
			this.addAlert('error', "Service URL is undefined, please check service paths passed to axios.");
			return undefined;
		}
```

You repeat a lot of logic here.

Reply:
You are right, I did the removal of the doubled logic.